tsup.config.ts

```diff
@@ -34,6 +34,7 @@ const COMMON_CONFIG: MandatoryOptions = {
   format: "cjs",
   outDir: "dist/common",
   platform: "node",
+  splitting: false,
```

Comment:
is this breaking current PR? if not, I would suggest we get it in another PR.
tbh, it is weird. Could you make sure it doesnt break dapps (can test locally against ans or some dapps).

Reply:
That's what's particularly weird, when I turn on splitting, it doesn't link some of the javascript underneath for common JS.  It works fine in ESM.

Reply:
Yes, exactly.

I can possibly go back through and poke through some pieces to figure out why it's happening, but I haven't had a huge amount of time to do it.

I suspect it's something small that's incompatible

Reply:
yea... last time it happened to me it was something with graphql codegen types, but doesnt seem this PR modifies something like that.

I can help take a look later

Reply:
bingo

https://github.com/aptos-labs/aptos-ts-sdk/pull/337